api/src/controllers/referent.js

```diff
@@ -310,21 +310,16 @@ router.put("/young/:id", passport.authenticate("referent", { session: false, fai
       // disable the 08 jun 21
       // await assignNextYoungFromWaitingList(young);
     }
-
-    // if a referent said that a young is present in the cohesion stay, we validate its phase1
     if (newYoung.cohesionStayPresence === "true" && young.statusPhase1 !== "DONE") {
```

Comment:
je mettrais ca, peu importe le statut de la phase 1 global
```suggestion
    if (newYoung.cohesionStayPresence === "true" && young.cohesionStayPresence !== "true") {
```